src/Vlingo.Actors/Plugin/Mailbox/AgronaMPSCArrayQueue/ManyToOneConcurrentArrayQueueMailbox.cs

```diff
@@ -42,13 +42,12 @@ public bool IsDelivering
 
         public bool IsPreallocated => false;
 
-        public int PendingMessages 
-            => throw new NotSupportedException("ManyToOneConcurrentArrayQueueMailbox does not support this operation.");
+        public int PendingMessages => queue.Count;
 
         public bool IsSuspended => false;
 
         public void Resume(string name) 
-            => throw new NotSupportedException("ManyToOneConcurrentArrayQueueMailbox does not support this operation.");
+            => Console.WriteLine($"WARNING: ManyToOneConcurrentArrayQueueMailbox does not support resume(): {name}");
```

Comment:
Shouldn't be logger used here?

Reply:
To pass a logger as a dependency in the mailbox, I will have to discuss it in detail. It is true actors will have their own mailboxes. But, I am just not sure how it would work in a distributed environment. Will discuss with @VaughnVernon and come back to it, if logging is required.